### PR TITLE
GitKey works for all tables now

### PR DESCRIPTION
**Pull request by fabiansinz**

### CODE REVIEW

File: djaddon/git.py [hunk 1]
```diff
@@ -9,9 +9,10 @@
 def _log_git_status(func):
     @wraps(func)
     def with_logging(*args, **kwargs):
-        ret = func(*args, **kwargs)
-        for key in (args[0] - args[0].GitKey()).project().fetch.as_dict:
-            args[0].GitKey().log_key(key)
+        with dj.conn().transaction:
```
Comment (eywalker) [inline djaddon/git.py]: Explicit calling of `dj.conn()` here is problematic - in the (near) future, we want to support multiple connections, thus we need to ensure that we minimize the reference to global connection fetcher like `dj.conn()` call. I believe that the first argument to the `with_logging` is `self`, so we should write `with_logging(self, *args, **kwargs)` and then replace the call to `dj.conn()` with `self.connection`

Comment (fabiansinz) [inline djaddon/git.py]: Good point. Will change that ....

**Comment by eywalker** [comment]
Travis just migrated their Precise Ubuntu build, which is screwing with the `hdf5` package. Let's actually modify the `.travis.yml` to make use of their containerized build environment (https://docs.travis-ci.com/user/migrating-from-legacy/?utm_source=legacy-notice&utm_medium=banner&utm_campaign=legacy-upgrade), and use the package `libhdf5-dev` instead.

**Comment by fabiansinz** [comment]
`libhdf5-dev` does not work. I try to figure out why.

**Comment by fabiansinz** [comment]
Alright. New `travis.yml` works!

**Comment by eywalker** [comment]
Yay!

**Pull request merged**